lib/gh-badges.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const { makeBadge } = require('./make-badge')
+const { PDFKitTextMeasurer, QuickTextMeasurer } = require('./text-measurer')
+
+class BadgeFactory {
+  constructor({ fontPath, fallbackFontPath, cache = true }) {
```

Comment:
`QuickTextMeasurer` doesn't cache as much as it memoizes the most common inputs. Something like `precomputeWidths` would be more accurate.

Reply:
Changed. Do you think this should default to `true` or `false`?

Reply:
`false`, since I’d wager the most common use will be generating a single badge.